README.md

```diff
@@ -101,9 +101,39 @@ python manage.py compile_pdfs --all_documents
 python manage.py compile_pdfs
 ```
 
+## AWS Buckets
```

Comment:
Is this a good idea? Maybe we should send to another bucket?

Reply:
I created an S3 bucket for testing (`datamade-metro-pdf-merger-testing`), and I documented that in the README.